index.js

```diff
@@ -24,67 +30,72 @@ function getRandomNumber(min, max) {
 
 function checkGuess() {
   // Get value from guess input element
-  const guess = parseInt(guessInput.value, 10);
-  attempts = attempts + 1;
+  let guess = Number(guessInput.value, 10);
+ 
 
   hideAllMessages();
 
+
   if (guess === targetNumber) {
-    numberOfGuessesMessage.style.display = '';
+    numberOfGuessesMessage.style.display = "";
     numberOfGuessesMessage.innerHTML = `You made ${attempts} guesses`;
 
-    correctMessage.style.display = '';
+    correctMessage.style.display = "";
+    correctMessage.innerHTML = `Congratulations, You guessed correctly! <br> Would you like to play again?`;
 
     submitButton.disabled = true;
     guessInput.disabled = true;
   }
 
   if (guess !== targetNumber) {
     if (guess < targetNumber) {
-      tooLowMessage.style.display = '';
+      tooLowMessage.style.display = "";
     } else {
-      tooLowMessage.style.display = '';
+      tooHighMessage.style.display = "";
     }
+    attempts = attempts + 1;
```

Comment:
I would move this back up to the top of the function. Otherwise, the number of guesses will be off in some scenarios. For example, if you get the guess right the first time it will say, "You made 0 guesses!" even though you made one. Basically, you want to increment your guesses before you run any logic that will print to the DOM so your number is correct.